autoload/vundle/installer.vim

```diff
@@ -443,6 +443,20 @@ func! s:sync(bang, bundle) abort
     return 'error'
   end
 
+  if !exists('g:vundle_no_deps') || !g:vundle_no_deps
+    let deps = vundle#scripts#getdeps(a:bundle)
+    if !empty(deps)
+      for dependency in deps
+        if !has_key(g:bundle_names, dependency)
+          let newbundle = vundle#config#bundle(dependency)
```

Comment:
Please add a log entry to indicate this new plugin will be added as a dependency of the current plugin.